### PR TITLE
add a command that returns the number of connections per credential

### PR DESCRIPTION
**Pull request by coreypurcell**
```
 % heroku pg:user_connections --app APP postgresql-database-name
Credential      Connections
──────────────  ───────────
ua7almfsv0d8tq  24
```

The DUX team suggested that we create the new command in pg-extras first before we consider adding it to pg proper.

### CODE REVIEW

**Comment by camillebaldock** [comment]
Today is another one of those sad days where I am reminded that we never wrote any tests for this module...

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg-extras",
   "description": "A Heroku CLI plugin for awesome pg:* commands that are also great and fun and super.",
-  "version": "1.0.11",
+  "version": "1.0.12",
```
Comment (camillebaldock) [inline package.json]: so this version bump doesn't magically work via code, releasing a new version is documented in https://devcenter.heroku.com/articles/developing-cli-plugins#releasing-plugins; so i wouldn't touch the version here, but go through this devcenter article's instructions after merging to create a new release

File: Readme.md [hunk 2]
```diff
@@ -253,6 +253,13 @@ $ heroku pg:vacuum-stats
 $ heroku pg:mandelbrot
 ```
 
+```
+$ heroku pg:user-connections
+Credential      Connections
+──────────────  ───────────
+ua7almfsv0d8tq  24
```
Comment (camillebaldock) [inline Readme.md]: so there is a consistency question here potentially with `heroku pg:credentials` which lists the default credential as `default` and not the username. I feel we should stay consistent, and list the credential as `default` here too, any thoughts @uhoh-itsmaciek @coreypurcell ?

Comment (coreypurcell) [reply Readme.md]: consistency is probably good. Looks like I can see if the user is default by hitting the credentials endpoint.

Comment (coreypurcell) [reply Readme.md]: @camillebaldock I changed it to return default if it is the default credential

**Pull request merged**